Tests/ContainerTest.php

```diff
@@ -22,13 +24,18 @@
 
 class ContainerTest extends TestCase
 {
-    public static function setUpBeforeClass(): void
+    public function testContainerWithDbalOnly(): void
```

Comment:
How come is this not covered by existing tests already?

Reply:
Testing with only the DBAL? You tell me I don't know :sweat_smile:

Reply:
I realized now that ORM doesn't suppport DBAL 3.x yet, so this is why DBAL 3.x integration cannot be tested with ORM combination by existing tests currently.